docs/usage.md

```diff
@@ -188,6 +189,10 @@ Please note the following requirements:
 2. The metadata file has to follow the [QIIME2 specifications](https://docs.qiime2.org/2019.10/tutorials/metadata/)
 3. In case of multiple sequencing runs, specific naming of samples are required, see [here](#--multipleSequencingRuns)
 
+### `--qiime_timezone`
```

Comment:
Why not just call this parameter "timezone"?

Reply:
It's easier for us to track this back when naming it like this - not sure what you are doing in QIIME2 in the future, so we know why and where this came from initially 👍 E.g. when you at some point decide to remove this functionality, we might just upgrade QIIME2 and then drop the parameter again.

Reply:
Sure, I totally get that, but the solution proposed here changes the entire env's timezone --- this parameter name implies that in only impacts QIIME 2.

Reply:
True point - maybe @d4straub can note that in the docs at least :+1:

Reply:
Thanks for your suggestion @thermokarst, I updated the docs.